Auto update failing for tgpt

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [tgpt](https://github.com/termux/termux-packages/blob/f66705f4022a15ab7ea41c284055865f1421b10e/packages/tgpt) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating tgpt [Current version: 2.10.0]
INFO: package being updated to 2.10.1.
Downloading https://github.com/aandrew-me/tgpt/archive/refs/tags/v2.10.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 48843    0 48843    0     0  82319      0 --:--:-- --:--:-- --:--:-- 82319

You are about to commit these changes:

--------------------
diff --git a/packages/tgpt/build.sh b/packages/tgpt/build.sh
index 1be20d8ca1..11986367d4 100644
--- a/packages/tgpt/build.sh
+++ b/packages/tgpt/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/aandrew-me/tgpt
 TERMUX_PKG_DESCRIPTION="AI Chatbots in terminal without needing API keys"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2.10.0"
+TERMUX_PKG_VERSION="2.10.1"
 TERMUX_PKG_SRCURL=https://github.com/aandrew-me/tgpt/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=0e312176908d581eeb7f0df8fcd0524a4aa4702029d50f553f0f75d6c15bc0d9
+TERMUX_PKG_SHA256=a0fc596fd2b8aabac222e22e96fc02709e185031e31a15b522f4c5c59db892d9
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
--------------------

bump(main/tgpt): 2.10.1


Not committing to Git!

INFO: Trying to build package.
INFO: Package tgpt exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building tgpt for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13133  100 13133    0     0   122k      0 --:--:-- --:--:-- --:--:--  123k
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13319  100 13319    0     0  23427      0 --:--:-- --:--:-- --:--:-- 23448
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   1479      0 --:--:-- --:--:-- --:--:--  1482
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Wed 09 Jul 2025 02:00:00 AM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13119  100 13119    0     0  22324      0 --:--:-- --:--:-- --:--:-- 22349
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   1481      0 --:--:-- --:--:-- --:--:--  1479
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   1481      0 --:--:-- --:--:-- --:--:--  1479
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Tue 08 Jul 2025 12:36:02 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   1473      0 --:--:-- --:--:-- --:--:--  1471
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   1472      0 --:--:-- --:--:-- --:--:--  1471
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Tue 08 Jul 2025 09:52:52 PM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]:  12 1688k   12  211k    0     0   170k      0  0:00:09  0:00:01  0:00:08  170k
[packages-cf.termux.dev-apt-termux-main]: 100 1688k  100 1688k    0     0   968k      0  0:00:01  0:00:01 --:--:--  967k
::endgroup::
Downloading tgpt@2.10.1 source from 'https://github.com/aandrew-me/tgpt/archive/refs/tags/v2.10.1.tar.gz' if necessary...
Downloading https://github.com/aandrew-me/tgpt/archive/refs/tags/v2.10.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 48843    0 48843    0     0  95015      0 --:--:-- --:--:-- --:--:-- 95015
go: downloading github.com/c-bata/go-prompt v0.2.6
go: downloading github.com/fatih/color v1.18.0
go: downloading github.com/charmbracelet/bubbletea v1.2.4
go: downloading github.com/atotto/clipboard v0.1.4
go: downloading github.com/charmbracelet/bubbles v0.20.0
go: downloading github.com/olekukonko/ts v0.0.0-20171002115256-78ecb04241c0
go: downloading github.com/bogdanfinn/fhttp v0.5.32
go: downloading github.com/bogdanfinn/tls-client v1.7.11
go: downloading golang.design/x/clipboard v0.7.1
go: downloading github.com/charmbracelet/lipgloss v1.0.0
go: downloading github.com/charmbracelet/x/ansi v0.8.0
go: downloading github.com/charmbracelet/x/term v0.2.1
go: downloading github.com/muesli/ansi v0.0.0-20230316100256-276c6243b2f6
go: downloading github.com/muesli/cancelreader v0.2.2
go: downloading github.com/muesli/termenv v0.15.2
go: downloading github.com/bogdanfinn/utls v1.6.3
go: downloading github.com/tam7t/hpkp v0.0.0-20160821193359-2b70b4024ed5
go: downloading golang.org/x/net v0.34.0
go: downloading github.com/pkg/term v1.2.0-beta.2
go: downloading github.com/andybalholm/brotli v1.1.1
go: downloading github.com/klauspost/compress v1.17.11
go: downloading golang.org/x/mobile v0.0.0-20250606033058-a2a15c67f36f
go: downloading github.com/cloudflare/circl v1.5.0
go: downloading golang.org/x/crypto v0.32.0
go: downloading github.com/quic-go/quic-go v0.49.0
# golang.org/x/mobile/gl
In file included from ../../_cache/go-path/pkg/mod/golang.org/x/mobile@v0.0.0-20250606033058-a2a15c67f36f/gl/work.go:25:
./work.h:10:10: fatal error: 'GLES2/gl2.h' file not found
   10 | #include <GLES2/gl2.h>
      |          ^~~~~~~~~~~~~
1 error generated.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-07-09 02:04:10 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/16156960008">16156960008</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/tgpt/0001-disable-clipboard.patch
+++ b/packages/tgpt/0001-disable-clipboard.patch
@@ -1,0 +1,27 @@
+`golang.design/x/clipboard` requires JNI
+
+--- a/src/helper/helper.go
++++ b/src/helper/helper.go
+@@ -17,7 +17,6 @@
+ 	"github.com/aandrew-me/tgpt/v2/src/structs"
+ 	http "github.com/bogdanfinn/fhttp"
+ 	"github.com/fatih/color"
+-	"golang.design/x/clipboard"
+ 
+ 	"github.com/olekukonko/ts"
+ 
+@@ -774,14 +773,6 @@
+ 
+ 				if userInput == "y" || userInput == "" {
+ 					ExecuteCommand(ShellName, ShellOptions, fullText)
+-				} else {
+-					err := clipboard.Init()
+-
+-					if err == nil {
+-						clipboard.Write(clipboard.FmtText, []byte(fullText))
+-
+-						fmt.Println("Copied command to clipboard")
+-					}
+ 				}
+ 			}
+ 		}

--- a/packages/tgpt/build.sh
+++ b/packages/tgpt/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/aandrew-me/tgpt
 TERMUX_PKG_DESCRIPTION="AI Chatbots in terminal without needing API keys"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2.10.0"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="2.10.1"
 TERMUX_PKG_SRCURL=https://github.com/aandrew-me/tgpt/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=0e312176908d581eeb7f0df8fcd0524a4aa4702029d50f553f0f75d6c15bc0d9
+TERMUX_PKG_SHA256=a0fc596fd2b8aabac222e22e96fc02709e185031e31a15b522f4c5c59db892d9
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
@@ -14,7 +13,7 @@ termux_step_pre_configure() {
 }
 
 termux_step_make() {
-	go build
+	go build -trimpath -ldflags="-s -w"
 }
 
 termux_step_make_install() {

